run.sh

```diff
@@ -111,7 +111,7 @@ Options
                                   specified multiple times. The valid values for REPOSITORY are only 'cassandra' and
                                   'cassandra-website'. The values for TAGS are split by comma with no spaces. The
                                   repository and list of tags are split by a colon ':'. For example:
-                                    -t cassandra:cassandra-4.0,cassandra-3.11,cassandra-3.0
+                                    -t cassandra:cassandra-4.0,cassandra-3.11
```

Comment:
```suggestion
                                    -t cassandra:trunk,cassandra-4.0,cassandra-3.11
```
```suggestion
                                    -t cassandra:cassandra-4.0,cassandra-3.11
```

Reply:
The `-t` option is for tags on a repository. Does the Cassandra project have a tag called `trunk` as well?

Reply:
ah yes, i was thinking of the `-b …` option. 

Those tags will not work atm. (They won't work until from the releases that happen after Lorina's branches are merged.)